viewer/src/app/feature-view/feature-view.component.ts

```diff
@@ -167,39 +186,49 @@ export class FeatureViewComponent implements OnChanges, AfterViewInit {
     this.map.setView(view)
   }
 
-  brtLayer() {
-    const projectionExtent = [-285401.92, 22598.08, 595401.9199999999, 903401.9199999999]
-    const projection = new Projection({
-      code: 'EPSG:28992',
-      units: 'm',
-      extent: projectionExtent,
-    })
-    const resolutions = [3440.64, 1720.32, 860.16, 430.08, 215.04, 107.52, 53.76, 26.88, 13.44, 6.72, 3.36, 1.68, 0.84, 0.42, 0.21]
-    //const size = ol.extent.getWidth(projectionExtent) / 256
-
-    const matrixIds = []
-    for (let i = 0; i < resolutions.length; ++i) {
-      matrixIds[i] = 'EPSG:28992:' + i
-    }
+  brtLayer(p: { projection: Projection; resolutions: number[]; matrixIds: string[] }) {
     return new Tile({
       source: new WMTSSource({
         attributions: 'Kaartgegevens: &copy; <a href="https://www.kadaster.nl">Kadaster</a>',
         url: environment.bgtBackgroundUrl,
         layer: 'grijs',
-        matrixSet: 'EPSG:28992',
+        matrixSet: p.projection.getCode(),
         format: 'image/png',
-        projection: projection,
+        projection: p.projection,
         tileGrid: new WMTSTileGrid({
-          origin: getTopLeft(projectionExtent),
-          resolutions: resolutions,
-          matrixIds: matrixIds,
+          origin: getTopLeft(p.projection.getExtent()),
+          resolutions: p.resolutions,
+          matrixIds: p.matrixIds,
         }),
         style: 'default',
         wrapX: false,
       }),
     })
   }
 
+  zzprojectionSet25831() {
+    const europ = new Projection({
```

Comment:
europ -> europe. Maar is `etrs89` niet een betere term?

Reply:
✔️ refactored (functie verwijderd) .